src/runtime/providers/ipx.ts

```diff
@@ -12,7 +12,7 @@ const operationsGenerator = createOperationsGenerator({
     quality: 'q',
     background: 'b'
   },
-  joinWith: ',',
+  joinWith: '_',
```

Comment:
Comma support should be supported in upstream nitropack version (today/tomorrow with Nuxt 3.1). More info https://github.com/unjs/nitro/pull/799